src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -32,24 +32,24 @@
 abstract class DoctrineType extends AbstractType
 {
     /**
-     * @var ManagerRegistry
+     * @var DoctrineChoiceLoader[]
      */
-    protected $registry;
+    private static $choiceLoaders = array();
```

Comment:
This is not a problem. The object manager hash is included in the cache keys, so entities assigned to separate object managers will naturally have different cache keys.